src/MHTimer.js

```diff
@@ -1360,7 +1368,12 @@ function addRemind(timerRequest, message) {
             ? `\`\`\`${responses.join('\n')}\`\`\``
             : `I couldn't find a matching reminder for you in '${requestName}'.`,
         );
-        return;
+        if(responses.length) {
```

Comment:
you can simplify this as `return CommandData(message, true, responses.length > 0);`